rmgpy/data/thermo.py

```diff
@@ -2445,7 +2445,7 @@ def _add_ring_correction_thermo_data_from_tree(self, thermo_data, ring_database,
             entry = ring_database.descend_tree(molecule, atoms)
             matched_ring_entries.append(entry)
 
-        if matched_ring_entries is []:
+        if matched_ring_entries == []:
```

Comment:
Why do we not use `if not matched_ring_entries:`? That should be the more Pythonic way to check if a list is empty.

Reply:
Good point